src/client/components/Home.js

```diff
@@ -1,5 +1,39 @@
-import React from 'react';
+import React, { Component } from 'react';
+// import { graphql } from 'react-apollo';
+// import gql from 'graphql-tag';
 
-const Home = props => <h1>Your Home</h1>;
+class Home extends Component {
+  render() {
+    // const { homeQuery } = this.props;
+    // if (homeQuery && homeQuery.loading) {
+    //   return <div>Loading...</div>;
+    // }
 
+    // if (homeQuery && homeQuery.error) {
+    //   return <div>Error</div>;
+    //   console.log(error);
+    // }
+
+    // return (
+    //   <div>
+    //     <h2>{homeQuery.home.name}</h2>
+    //     <p>{homeQuery.home.description}</p>
+    //   </div>
+    // );
+
+    return <div>hello</div>;
```

Comment:
Any reason for the comments in this file?

Reply:
Yes. It's how i think we will eventually display the home page info using that stuff but we dont have any user relationship with home set up yet so that's why it's commented out.

Reply:
@joshblack If it's better for it to be gone, let me know 😃. It makes sense if comments like that shouldn't be pushed to master.